typescript/typescript.vs2013.nuspec

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>typescript.vs2013</id>
+        <version>1.0.0</version>
+        <title>TypeScript.VS (Visual Studio 2013)</title>
+        <authors>Thomas Gassmann</authors>
```

Comment:
Author should be the original author of the software (Microsoft in this case). Add additional `<owners>IOZ AG</owners>` tag for documenting ownership.